hw/top_earlgrey/ip/ast/ast.core

```diff
@@ -51,11 +61,12 @@ targets:
           - "-Wall"
 
   sim:
+    <<: *default_target  
     default_tool: vcs
     filesets:
       - files_rtl
     tools:
       vcs:
-        vcs_options: [-timescale=1ns/1ps]
+        vcs_options: [-timescale=1ns/1ps -l vcs.log]
```

Comment:
The vcs_options key expects an array of command-line arguments, so please convert this to the following syntax to ensure that FuseSoC can properly quote the arguments and pass them on.

```suggestion
        vcs_options:
          - "-timescale=1ns/1ps"
          - "-l"
          - "vcs.log"
```

Alternatively, the [] syntax works as well, but is less recommended. If you want to stick with it, the syntax would be

```suggestion
        vcs_options: ["-timescale=1ns/1ps", "-l", "vcs.log"]
```

See also https://fusesoc.readthedocs.io/en/master/user/build_system/core_files.html#things-one-should-know-about-yaml for a short explanation of the (YAML) syntax that is used here.

Reply:
Are you running VCS via Dvsim to simulate AST? 
If so, this may not be the right place to add these options.

@sriyerg, where should these be placed exactly?